targets/TARGET_GigaDevice/TARGET_GD32F4XX/us_ticker.c

```diff
@@ -0,0 +1,389 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2018 GigaDevice Semiconductor Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "gd32f4xx.h"
+#include "us_ticker_api.h"
+#include "PeripheralNames.h"
+#include "hal_tick.h"
+
+#if TICKER_TIMER_WIDTH_BIT == 16
+uint32_t time_before;
+uint32_t total_elapsed_time;
+#endif
+
+/* this variable is set to 1 at the end of mbed_sdk_init function.
+the ticker_read_us() function must not be called until the mbed_sdk_init is terminated */
+extern int mbed_sdk_inited;
+uint32_t ticker_timer_cnt;
```

Comment:
Are these used anywhere else (not defined as static) ? Github does not show me all files at once to search this